libs/Functions.php

```diff
@@ -534,10 +542,16 @@ public function post_type($icon=true){
 						$post_icon_img = '<i class="fas fa-thumbtack"></i>';
 					}
 				}
+			}else{
+				$post_icon_img = false;
 			}
 
-			return '<span class="meta-item post-type">' . $post_icon_img . ucwords(get_post_type()) . '</span>';
-		}
+			if( $data['linked'] ){
+				return '<span class="meta-item post-type"><a href="'.esc_url( get_post_type_archive_link( $post_type ) ).'" title="See all '.$post_type_labels->name.'">'.$post_icon_img.esc_html($post_type_labels->singular_name).'</a></span>';
```

Comment:
Same note about space around `.` here too. Just noting for sake of review completeness.